### DISCUSSION TITLE
Server Certificate Verify Failed

### DISCUSSION THREAD

**Discussion by NCVito** [yuezk/GlobalProtect-openconnect#217] [Q&A]
When connecting to a VPN I see:

Certificate from VPN server "xxx.xxx.xxx.xx" failed verification.
Reason: signer not found
To trust this server in future, perhaps add this to your command line:
    --servercert pin-sha256:xxxxxxxxxxxxxxxxxxxx
Enter 'yes' to accept, 'no' to abort; anything else to view:

However, it doesn't accept any input from me. Is there a way for me to force it to accept untrusted certs?

**Comment by yuezk** [comment]
@NCVito You can configure your certificate in the configuration file. See https://github.com/yuezk/GlobalProtect-openconnect/wiki/Configuration#example
